packages/react-components/src/molecules/LabeledTypeahead.tsx

```diff
@@ -0,0 +1,21 @@
+import { ComponentProps, ReactNode } from 'react';
+import { Typeahead, Paragraph, Label } from '../atoms';
+import { perRem } from '../pixels';
+
+type LabeledTypeaheadProps = {
+  readonly title: ReactNode;
+} & Exclude<ComponentProps<typeof Typeahead>, 'id'>;
+const LabeledTypeahead: React.FC<LabeledTypeaheadProps> = ({
+  title,
+  ...typeaheadProps
+}) => (
+  <div css={{ paddingBottom: `${18 / perRem}em` }}>
+    <Label forContent={(id) => <Typeahead {...typeaheadProps} id={id} />}>
```

Comment:
Do we need the required thing?

Reply:
What required thing, sorry?

Reply:
Clarified, was about the asterisk, but it's there already

Reply:
pushed an additional refactor commit to straighten out asterisk handling across input types